CMakeLists.txt

```diff
@@ -26,6 +26,35 @@ endif()
 
 file(GLOB _files *.fdm_material *.sig)
 
+# Determine version number
+find_package(Git)
+
+set(GIT_VERSION "Unknown")
```

Comment:
You don't need to define this. You can use `GIT_VERSION_STR` if `Git` is found. See https://cmake.org/cmake/help/v3.12/module/FindGit.html